<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=gb2312">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List href="help.files/filelist.xml">
<link rel=Edit-Time-Data href="help.files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Windows 用户</o:Author>
  <o:LastAuthor>Windows 用户</o:LastAuthor>
  <o:Revision>1</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:Created>2016-10-12T11:14:00Z</o:Created>
  <o:LastSaved>2016-10-12T11:15:00Z</o:LastSaved>
  <o:Pages>10</o:Pages>
  <o:Words>2485</o:Words>
  <o:Characters>14170</o:Characters>
  <o:Lines>118</o:Lines>
  <o:Paragraphs>33</o:Paragraphs>
  <o:CharactersWithSpaces>16622</o:CharactersWithSpaces>
  <o:Version>15.00</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="help.files/themedata.thmx">
<link rel=colorSchemeMapping href="help.files/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:SpellingState>Clean</w:SpellingState>
  <w:GrammarState>Clean</w:GrammarState>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:DrawingGridVerticalSpacing>7.8 磅</w:DrawingGridVerticalSpacing>
  <w:DisplayHorizontalDrawingGridEvery>0</w:DisplayHorizontalDrawingGridEvery>
  <w:DisplayVerticalDrawingGridEvery>2</w:DisplayVerticalDrawingGridEvery>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>ZH-CN</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:SpaceForUL/>
   <w:BalanceSingleByteDoubleByteWidth/>
   <w:DoNotLeaveBackslashAlone/>
   <w:ULTrailSpace/>
   <w:DoNotExpandShiftReturn/>
   <w:AdjustLineHeightInTable/>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
   <w:UseFELayout/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="false"
  DefSemiHidden="false" DefQFormat="false" DefPriority="99"
  LatentStyleCount="371">
  <w:LsdException Locked="false" Priority="0" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 9"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 9"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footnote text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="header"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footer"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index heading"/>
  <w:LsdException Locked="false" Priority="35" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="table of figures"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="envelope address"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="envelope return"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footnote reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="line number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="page number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="endnote reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="endnote text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="table of authorities"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="macro"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="toa heading"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 5"/>
  <w:LsdException Locked="false" Priority="10" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Closing"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Signature"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="true"
   UnhideWhenUsed="true" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Message Header"/>
  <w:LsdException Locked="false" Priority="11" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Salutation"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Date"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text First Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text First Indent 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Note Heading"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Block Text"/>
  <w:LsdException Locked="false" Priority="0" SemiHidden="true"
   UnhideWhenUsed="true" Name="Hyperlink"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="FollowedHyperlink"/>
  <w:LsdException Locked="false" Priority="22" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Document Map"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Plain Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="E-mail Signature"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Top of Form"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Bottom of Form"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal (Web)"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Acronym"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Address"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Cite"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Code"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Definition"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Keyboard"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Preformatted"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Sample"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Typewriter"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Variable"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal Table"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation subject"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="No List"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Contemporary"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Elegant"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Professional"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Subtle 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Subtle 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Balloon Text"/>
  <w:LsdException Locked="false" Priority="39" Name="Table Grid"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Theme"/>
  <w:LsdException Locked="false" SemiHidden="true" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" SemiHidden="true" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" QFormat="true"
   Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" QFormat="true"
   Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" QFormat="true"
   Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" QFormat="true"
   Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" QFormat="true"
   Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" QFormat="true"
   Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" SemiHidden="true"
   UnhideWhenUsed="true" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="TOC Heading"/>
  <w:LsdException Locked="false" Priority="41" Name="Plain Table 1"/>
  <w:LsdException Locked="false" Priority="42" Name="Plain Table 2"/>
  <w:LsdException Locked="false" Priority="43" Name="Plain Table 3"/>
  <w:LsdException Locked="false" Priority="44" Name="Plain Table 4"/>
  <w:LsdException Locked="false" Priority="45" Name="Plain Table 5"/>
  <w:LsdException Locked="false" Priority="40" Name="Grid Table Light"/>
  <w:LsdException Locked="false" Priority="46" Name="Grid Table 1 Light"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark"/>
  <w:LsdException Locked="false" Priority="51" Name="Grid Table 6 Colorful"/>
  <w:LsdException Locked="false" Priority="52" Name="Grid Table 7 Colorful"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 1"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 1"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 1"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 2"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 2"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 2"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 3"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 3"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 3"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 4"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 4"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 4"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 5"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 5"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 5"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 6"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 6"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 6"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="46" Name="List Table 1 Light"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark"/>
  <w:LsdException Locked="false" Priority="51" Name="List Table 6 Colorful"/>
  <w:LsdException Locked="false" Priority="52" Name="List Table 7 Colorful"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 1"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 1"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 1"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 2"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 2"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 2"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 3"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 3"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 3"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 4"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 4"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 4"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 5"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 5"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 5"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 6"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 6"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 6"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 6"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:宋体;
	panose-1:2 1 6 0 3 1 1 1 1 1;
	mso-font-alt:SimSun;
	mso-font-charset:134;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:3 680460288 22 0 262145 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1107305727 0 0 415 0;}
@font-face
	{font-family:"\@宋体";
	panose-1:2 1 6 0 3 1 1 1 1 1;
	mso-font-charset:134;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:3 680460288 22 0 262145 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	mso-pagination:none;
	font-size:10.5pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:宋体;
	mso-font-kerning:1.0pt;}
a:link, span.MsoHyperlink
	{mso-style-unhide:no;
	mso-style-parent:"";
	mso-ansi-font-size:10.0pt;
	font-family:"Arial",sans-serif;
	mso-ascii-font-family:Arial;
	mso-hansi-font-family:Arial;
	color:navy;
	text-decoration:none;
	text-underline:none;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:#954F72;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
span.SpellE
	{mso-style-name:"";
	mso-spl-e:yes;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:"Calibri",sans-serif;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
 /* Page Definitions */
 @page
	{mso-page-border-surround-header:no;
	mso-page-border-surround-footer:no;}
@page WordSection1
	{size:595.3pt 841.9pt;
	margin:72.0pt 90.0pt 72.0pt 90.0pt;
	mso-header-margin:42.55pt;
	mso-footer-margin:49.6pt;
	mso-paper-source:0;
	layout-grid:15.6pt;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 @list l0
	{mso-list-id:712733282;
	mso-list-template-ids:712733282;}
@list l0:level1
	{mso-level-tab-stop:18.0pt;
	mso-level-number-position:left;
	margin-left:18.0pt;
	text-indent:-18.0pt;}
@list l0:level2
	{mso-level-number-format:alpha-lower;
	mso-level-text:"%2\)";
	mso-level-tab-stop:42.0pt;
	mso-level-number-position:left;
	margin-left:42.0pt;
	text-indent:-21.0pt;}
@list l0:level3
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:63.0pt;
	mso-level-number-position:right;
	margin-left:63.0pt;
	text-indent:-21.0pt;}
@list l0:level4
	{mso-level-tab-stop:84.0pt;
	mso-level-number-position:left;
	margin-left:84.0pt;
	text-indent:-21.0pt;}
@list l0:level5
	{mso-level-number-format:alpha-lower;
	mso-level-text:"%5\)";
	mso-level-tab-stop:105.0pt;
	mso-level-number-position:left;
	margin-left:105.0pt;
	text-indent:-21.0pt;}
@list l0:level6
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:126.0pt;
	mso-level-number-position:right;
	margin-left:126.0pt;
	text-indent:-21.0pt;}
@list l0:level7
	{mso-level-tab-stop:147.0pt;
	mso-level-number-position:left;
	margin-left:147.0pt;
	text-indent:-21.0pt;}
@list l0:level8
	{mso-level-number-format:alpha-lower;
	mso-level-text:"%8\)";
	mso-level-tab-stop:168.0pt;
	mso-level-number-position:left;
	margin-left:168.0pt;
	text-indent:-21.0pt;}
@list l0:level9
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:189.0pt;
	mso-level-number-position:right;
	margin-left:189.0pt;
	text-indent:-21.0pt;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:普通表格;
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.5pt;
	mso-bidi-font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-font-kerning:1.0pt;}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1026"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=ZH-CN link=navy vlink="#954F72" style='tab-interval:21.0pt;
text-justify-trim:punctuation'>

<div class=WordSection1 style='layout-grid:15.6pt'>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:
150%;mso-list:l0 level1 lfo1;tab-stops:list 18.0pt'><a name="OLE_LINK2"></a><a
name="OLE_LINK1"><span style='mso-bookmark:OLE_LINK2'><![if !supportLists]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;mso-fareast-font-family:
"Times New Roman";color:blue'><span style='mso-list:Ignore'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;color:blue'>What is </span></span></a><span
class=SpellE><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:14.0pt;line-height:150%;
color:blue'>MTGIpick</span></span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:14.0pt;
line-height:150%;color:blue'>?<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>Genomic <span
class=GramE>island</span> (GI) is a cluster of genes in prokaryotic genomes
that have probable horizontal origins. These genetic elements have been
associated with rapid adaptations in prokaryotes that are of medical,
economical or environmental importance, such as pathogen virulence, antibiotic
resistance, symbiotic interactions, and notable secondary metabolic
capabilities. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span class=SpellE><span lang=EN-US style='font-size:12.0pt;
line-height:150%'>MTGIpick</span></span></span></span><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span lang=EN-US
style='font-size:12.0pt;line-height:150%'> implements a multiscale statistical
algorithm to predict GIs from a single genome. It uses a small-scale test with large-scale
features to score small region deviating from the host and a large-scale
statistical test with small-scale features to identify multi-window segments
for identification of GIs. <span class=SpellE>MTGIpick</span> can identify GIs
from a single genome, without annotated information of genomes or prior
knowledge from other datasets. In simulations with alien fragments from
artificial and real genomes, <span class=SpellE>MTGIpick</span> reported robust
results across different experiments. From real biological data, <span
class=SpellE>MTGIpick</span> demonstrated better performance compared with
existing methods, and identified GIs with more accurate size.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span class=SpellE><span lang=EN-US style='font-size:12.0pt;
line-height:150%'>MTGIpick</span></span></span></span><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span lang=EN-US
style='font-size:12.0pt;line-height:150%'> was written in <span class=SpellE>Matlab</span>
and Java, compiled in Windows and Linux, and run on those platforms. We have
supplied a version of <span class=SpellE>MTGIpick</span>. The output of <span
class=SpellE>MTGIpick</span> consists of genomic signatures, conserved score of
each predicted GIs and the total scale predict GIs. In addition, <span
class=SpellE>MTGIpick</span> provides a new interactive genome visualization
tool, which uses <span class=SpellE>zoomable</span> sunburst and sunburst
partition to represent predicted GIs with conserved scores along the whole
genome.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span class=SpellE><span lang=EN-US style='font-size:12.0pt;
line-height:150%'>MTGIpick</span></span></span></span><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span lang=EN-US
style='font-size:12.0pt;line-height:150%'> version 1.2.0 has been released with
some new features:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:27.75pt;mso-para-margin-left:1.5gd;
text-indent:-12.0pt;mso-char-indent-count:-1.0;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>- The input format follows
the standard <span class=SpellE>Fasta</span> format. Multiple DNA sequences are
supported. The file suffix is recommended to be .<span class=SpellE>fasta</span>
or .fa.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:39.75pt;mso-para-margin-left:1.5gd;
text-indent:-24.0pt;mso-char-indent-count:-2.0;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>- The uploading progress
is monitored using a progress bar.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:39.75pt;mso-para-margin-left:1.5gd;
text-indent:-24.0pt;mso-char-indent-count:-2.0;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>- New options to process
the input file is added.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:39.75pt;mso-para-margin-left:1.5gd;
text-indent:-24.0pt;mso-char-indent-count:-2.0;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>- <a name="OLE_LINK4">The
software efficiency has been improved</a>.<span style='color:red'><o:p></o:p></span></span></span></span></p>

<p class=MsoNormal style='margin-left:39.75pt;mso-para-margin-left:1.5gd;
text-indent:-24.0pt;mso-char-indent-count:-2.0;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>- Reduce parameter
selection on prediction.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:39.75pt;mso-para-margin-left:1.5gd;
text-indent:-24.0pt;mso-char-indent-count:-2.0;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>- Warning messages are
added to notify the users when anything wrong occurs.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:39.75pt;mso-para-margin-left:1.5gd;
text-indent:-24.0pt;mso-char-indent-count:-2.0;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>- <a name="OLE_LINK7"></a><a
name="OLE_LINK6"><span style='mso-bookmark:OLE_LINK7'>Output file is more
readable and<span style='color:red'> </span>more details</span></a><span
style='mso-bookmark:OLE_LINK7'>.<o:p></o:p></span></span></span></span></p>

<span style='mso-bookmark:OLE_LINK7'></span>

<p class=MsoNormal style='margin-left:39.75pt;mso-para-margin-left:1.5gd;
text-indent:-24.0pt;mso-char-indent-count:-2.0;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>- An interactive
visualization tool is added for predicted genomic islands along the whole
genome.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:
150%;mso-list:l0 level1 lfo1;tab-stops:list 18.0pt'><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><![if !supportLists]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;mso-fareast-font-family:
"Times New Roman";color:blue'><span style='mso-list:Ignore'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;color:blue'>Functions and
features<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>The
recent development of GI detection methods has led to significant advances in
our understanding of microbial evolution and function. Despite these advances,
several challenges still exist regarding the use of multiscale features,
detection algorithm and boundary identification. <a name="OLE_LINK8">To solve
these problems, we reported a novel method called &quot;</a><span class=SpellE><span
style='mso-bookmark:OLE_LINK8'>MTGIpick</span></span><span style='mso-bookmark:
OLE_LINK8'>&quot;</span>, the first multiscale statistical test for the
identification of GIs. Several features were described below:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>1.
IST-LFS: an iteration of small-scale t-test with large-scale feature selection
to quantifying the compositional difference between a region and the ‘native’
genome regions.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>2. MSA: A
multiscale segmentation algorithm to investigate the variability of genomic
signals and to identify large, multi-window segments.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>3.
CG-MJSD: A boundary detection method based on CG-based segmentation and
Markovian Jensen-Shannon divergence.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>4. The
methods and parameters could be chosen according to your purpose.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>5. Using <span
class=SpellE>Zoomable</span> Sunburst and Sunburst Partition to represent the
predicted GIs with conserved scores along the whole genome.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:
150%;mso-list:l0 level1 lfo1;tab-stops:list 18.0pt'><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><![if !supportLists]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;mso-fareast-font-family:
"Times New Roman";color:blue'><span style='mso-list:Ignore'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;color:blue'>Install and Run<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'><span style='mso-spacerun:yes'>&nbsp; </span><span
style='mso-spacerun:yes'>&nbsp;</span><span class=SpellE>MTGIpick</span> has
been compiled and tested under Sun Java <span class=SpellE>interpretera</span>
and <span class=SpellE>Matlab</span>. <span class=SpellE>MTGIpick</span> can be
used in Windows- and Linux<span class=GramE>-<span
style='mso-spacerun:yes'>&nbsp; </span>platforms</span>. Java Virtual Machine
and MATLAB Compiler Runtime (MCR) are required for <span class=SpellE>MTGIpick</span>
setup on your platform. However, we strongly advise the use of <span
class=SpellE>openjdk</span> (JDK) instead of the Oracle version of java virtual
machine when working on <span class=SpellE>linux</span>-based machines as the
Oracle version may result in some exceptions during the analyses. All <span
class=SpellE>softwares</span> can be downloaded from our websites </span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-family:"Arial",sans-serif'>(</span></span></span><a
href="http://bioinfo.zstu.edu.cn/MTGI,"><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:10.0pt;
mso-bidi-font-size:12.0pt;line-height:150%;font-family:"Arial",sans-serif'>http://bioinfo.zstu.edu.cn/MTGI,</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'></span></span></a><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-family:"Arial",sans-serif'> </span></span></span><a
href="https://github.com/bioinfo0706/MTGIpick"><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:10.0pt;
mso-bidi-font-size:12.0pt;line-height:150%;font-family:"Arial",sans-serif'>https://github.com/bioinfo0706/MTGIpick</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'></span></span></a><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
class=GramE><span lang=EN-US style='font-family:"Arial",sans-serif'>) </span><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>.</span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'> <span class=SpellE>MTGIpick</span>
file contains its corresponding JDK and MCR on different platforms.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%;
color:blue'>3.1) Windows (Tested on win7 x64)<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'><span style='mso-spacerun:yes'>&nbsp;</span><span
style='mso-spacerun:yes'>&nbsp; </span>Install<span style='color:red'> </span>MCR
8.4 first, and<span style='color:red'> </span>run <span class=SpellE>MTGIpick</span>
setup directly.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%;
color:blue'>3.2) Linux (Tested on Centos7)<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>Step 1<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>To install
JDK 1.8 and MCR 8.1,<span style='color:red'> </span>it requires a simple
command line as follow:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>&gt; bash
install_jdk-mcr_linux.sh<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>Step 2<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>To run
the <span class=SpellE>MTGIpick</span> software, just type a simple command
line as follow:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>&gt;bash
run_MTGI_linux.sh<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:
150%;mso-list:l0 level1 lfo1;tab-stops:list 18.0pt'><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><![if !supportLists]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;mso-fareast-font-family:
"Times New Roman";color:blue'><span style='mso-list:Ignore'>4.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;color:blue'>Inputs<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:blue'>4.1) Input file format</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:14.0pt;line-height:150%;color:blue'><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'><span style='mso-spacerun:yes'>&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span> accepts DNA sequences, and the input file has to
be in <span class=SpellE>fasta</span>, fa, or <span class=SpellE>fna</span>
format. For example, the file name is <span class=SpellE>example.fasta</span>
or <span class=SpellE>example.fa</span>, its content looks like this:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>&gt;cya_GI1_93960_99000_GI2_208020_223020_GI3_408480_410520<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>CCCCATTCCCCCCATTCCCTCCTTTTCCACCATACCCTCTTTTCCCCTCGTTGCCCCCAA<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>ATTTTTACGCATTTCCCCATTAATGCGATGATCCCAGCGCGAAAGCATCTGTGATTAAGA<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>CGTCTATCAATTTATACTCGTTAGGGTTTTTTCTTCGGTGGTACCATCTGGGCGCCTACG<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>&gt;</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span style='font-size:9.0pt;line-height:150%;
font-family:宋体;mso-ascii-font-family:"Times New Roman";mso-hansi-font-family:
"Times New Roman"'>……</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>Example DNA input (.<span class=SpellE>fna</span> format),
its content looks like this:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>&gt;cya_GI1_93960_99000_GI2_208020_223020_GI3_408480_410520<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>CCCCATTCCCCCCATTCCCTCCTTTTCCACCATACCCTCTTTTCCCCTCGTTGCCCCCAA<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>ATTTTTACGCATTTCCCCATTAATGCGATGATCCCAGCGCGAAAGCATCTGTGATTAAGA<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>......<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>&gt;<span class=SpellE>ctt</span>_
GI2_208020_223020_GI1_93960_99000_GI3_408480_410520<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>CCCCATTCCCCCCATTCCCTCCTTTTCCACCATACCCTCTTTTCCCCTCGTTGCCCCCAA<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>ATTTTTACGCATTTCCCCATTAATGCGATGATCCCAGCGCGAAAGCATCTGTGATTAAGA<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:9.0pt;
line-height:150%'>......<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'><span style='mso-spacerun:yes'>&nbsp;&nbsp; </span><span
style='color:red'><span style='mso-spacerun:yes'>&nbsp;</span></span>Please
note <span class=SpellE>GenBank</span> (from NCBI) will not work with <span
class=SpellE>MTGIpick</span>. If using sequences from NCBI be sure to save them
as FASTA format first. Sequence format conversion tools are available at
http://www.ebi.ac.uk/Tools/sfc/. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:24.0pt;mso-char-indent-count:2.0;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>The
sequences can be uploaded to <span class=SpellE>MTGIpick</span> in a file. It
is very important that each of the sequences has a unique name. If they do not,
the software will fail. There must be no empty lines, white spaces or control
characters between sequences or at the top of the file. This will also cause
the software to fail. <span style='mso-spacerun:yes'>&nbsp;</span><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:blue'>4.2) Process Input file</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:14.0pt;line-height:150%;color:blue'><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:24.0pt;mso-char-indent-count:2.0;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>There
will be an upload progress bar to monitor upload progress when clicking the
button to upload a file. If the input file you uploaded contains at least two
sequences, a dialog box appears to tell you to select a way to process the
input file. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:24.0pt;mso-char-indent-count:2.0;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%;
color:blue'>Predict each sequence separately</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'>. For each DNA sequence of
input file, <span class=SpellE>MTGIpick</span> will process it separately and
predict its GIs.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:24.0pt;mso-char-indent-count:2.0;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%;
color:blue'>Assemble and predict</span></span></span><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span lang=EN-US
style='font-size:12.0pt;line-height:150%'>. The sequences from the input file
are assembled into a sequence according to the order of these sequences in the
input file, and <span class=SpellE>MTGIpick</span> will process it as a
sequence and predict its GIs. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:
150%;mso-list:l0 level1 lfo1;tab-stops:list 18.0pt'><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><![if !supportLists]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;mso-fareast-font-family:
"Times New Roman";color:blue'><span style='mso-list:Ignore'>5.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='font-size:14.0pt;line-height:150%;color:blue'>IST-LFS method<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:blue'>5.1) Framework<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>IST-LFS
is a proposed small-scale t-test with large-scale feature selection that was
used to quantify the compositional differences of a region from the host in the
<span class=SpellE>MTGIpick</span>. It is efficient at detecting horizontal
gene transfers or genomic islands with small sizes. The steps are described
below:<span style='color:blue'><o:p></o:p></span></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>a) Split
a genome into n non-overlapping windows of size 1kb.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>b)
Calculate the<span style='color:red'> </span><span style='color:black'>frequencies</span><span
style='color:red'> </span><span style='color:black'>o</span>f the <span
class=SpellE>tetranucleotides</span> in each window as genomic signatures.<a
name="OLE_LINK10"><o:p></o:p></a></span></span></span></p>

<span style='mso-bookmark:OLE_LINK10'></span>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>c)
Extract the signatures of the host with the help of the confidence intervals on
the windows’ variances.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>d)
Calculate the kurtosis of each <span class=SpellE>tetranucleotide</span> across
n windows and select the windows with a larger kurtosis as informative
signatures.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>e)
Measure the divergence of the <span class=SpellE>ith</span> window from the
host using the two-sample t-test.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>f) Select
windows whose scores are large enough to be considered to be statistically
significant. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>g) Delete
the selected windows and update all of windows of the genome; then, repeat
steps d-f until there is no window to be found.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>h) Refine
the boundaries of the predicted genomic islands using the CG-MJSD method.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:blue'>5.2) Parameters of IST-LFS Software<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Word size:</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'> the length of k-<span
class=SpellE>mer</span>, and default is 4.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Windowed
transform:</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'> the total number of the windows used in genomic
transformation, and default is 1.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Iteration
time: </span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'>the periods of time that are repeated to select windows whose
scores are large enough to be considered statistically significant, and default
is 5.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Core feature
size:</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'> the size of selected features by the proposed kurtosis, and
default is 256.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Eye window
size: </span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'>the size of eye windows used in the proposed divergence
measure based on two-sample t-test, and default is 2.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Time
standard error:</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'> the standard deviation of the mean of the window scores to
select windows associated with putative GIs, and default is 0.05.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Upstream/downstream
of</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span style='font-size:12.0pt;line-height:150%;
font-family:宋体;mso-ascii-font-family:"Times New Roman";mso-hansi-font-family:
"Times New Roman";color:#0000CC'>‘</span></span></span><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span lang=EN-US
style='font-size:12.0pt;line-height:150%;color:#0000CC'>raw</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
style='font-size:12.0pt;line-height:150%;font-family:宋体;mso-ascii-font-family:
"Times New Roman";mso-hansi-font-family:"Times New Roman";color:#0000CC'>’</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>GIs:</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'> <span class=GramE>the</span>
length of sequences around ‘raw’ GIs to refine the boundaries of predicted GIs,
and default is 5kb.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:
150%;mso-list:l0 level1 lfo1;tab-stops:18.0pt'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><![if !supportLists]><span lang=EN-US
style='font-size:14.0pt;line-height:150%;mso-fareast-font-family:"Times New Roman";
color:blue'><span style='mso-list:Ignore'>6.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span class=SpellE><span lang=EN-US
style='font-size:14.0pt;line-height:150%;color:blue'>MTGIpick</span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:14.0pt;line-height:150%;color:blue'> method<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:blue'>6.1) Framework<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span class=SpellE><span lang=EN-US style='font-size:12.0pt;
line-height:150%'>MTGIpick</span></span></span></span><span style='mso-bookmark:
OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span lang=EN-US
style='font-size:12.0pt;line-height:150%'> is a novel method for the robust
identification of GIs using the multiscale statistical testing. The steps are described
below:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>a) Split
a genome into n non-overlapping windows of size 1kb.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>b)
Calculate the frequencies of the <span class=SpellE>tetranucleotides</span> in
each window as genomic signatures.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>c) At a
smaller scale, we propose an iteration of small-scale t-tests with large-scale feature
selection (IST-LFS) to quantify the compositional differences of a region from
the host. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>d) At a
large scale, we investigated the variability of genomic signals and used
multiscale segmentation algorithm (MSA) to identify large, multi-window segments.
<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>e)
Calculate conserved score of each nucleotide according to the total number of
appearances in the selected segments, from which GIs are detected with respect
to their conserved scores.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%'>f) Refine
the boundaries of the predicted genomic islands using the CG-MJSD method.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:blue'>6.2) Parameters of <span class=SpellE>MTGIpick</span>
Software<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Word size:</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'> the length of k-<span
class=SpellE>mer</span>, and default is 4.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Windowed
transform:</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'> the total number of the windows used in genomic
transformation, and default is 4.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Iteration
time: </span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'>the periods of time that are repeated to select windows whose
scores are large enough to be considered statistically significant, and default
is 5.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Core feature
size:</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'> the size of selected features by the proposed kurtosis, and
default is 256.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Eye window
size: </span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'>the size of eye windows used in the proposed divergence
measure based on two-sample t-test, and default is 5.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Time
standard error:</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'> the standard deviation of the mean of the window scores to
select windows associated with putative GIs, and default is 0.05.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Total scale:</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'> the total number of the
scales in the multiscale segmentation algorithm, and default is 45.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Minimum GI
size:</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'> the minimum size of predicted genomic islands, and default
is 10kb. A smaller value is recommended if you want to predict GIs with small
size.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;line-height:150%'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;color:#0000CC'>Time
standard error:</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%'> the standard deviation of the mean of enrichment scores to
select segments associated with putative GIs, and default is 0.3.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='text-indent:18.0pt;mso-char-indent-count:1.5;
line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;line-height:150%;
color:#0000CC'>Upstream/downstream of ‘raw’ GIs:</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%'> the length of sequences
around ‘raw’ GIs to refine the boundaries<span style='color:red'> </span>of
predicted GIs, and default is 5kb.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:
150%;mso-list:l0 level1 lfo1;tab-stops:18.0pt'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><![if !supportLists]><span lang=EN-US
style='font-size:14.0pt;line-height:150%;mso-fareast-font-family:"Times New Roman";
color:blue'><span style='mso-list:Ignore'>7.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-US style='font-size:14.0pt;
line-height:150%;color:blue'>Results<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:blue'>7.1) Outputs<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp; </span>Once
your project is complete, a dialog box</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:red'> </span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;mso-bidi-font-family:Arial;color:black'>will appear to </span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'>tell you to select a way to download the results. There are two
ways to download the results:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp;&nbsp; </span>1). Download
the results by clicking save button.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp;&nbsp; </span>2). Find the
results in the same directory where the input file is stored.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp;&nbsp; </span>The outputs
of the <span class=SpellE>MTGIpick</span> consist of genomic signatures,
predicted GIs of each scale, predicted GIs of total scales, and GI
visualization. They are stored in the same directory where the input file is
stored. Output of the genomic signatures is a Zip file whose name is created by
the input file name and signature. If the input file contains at least two
sequences, the Zip file contains more signature files for all the sequences. <o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US>Example the genomic signature file, its content
looks like this:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp;&nbsp; </span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'>Example
the genomic signature file, its content looks like this:<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>&gt;gi|16758993|ref|NC_003198.1|
Salmonella <span class=SpellE>enterica</span> subsp. <span class=SpellE>enterica</span>
<span class=SpellE>serovar</span> <span class=SpellE>Typhi</span> str. CT18<span
style='mso-font-kerning:0pt'><o:p></o:p></span></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>NO<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>kmer</span><span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>kurtosis score<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>1<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>TCCC<span style='mso-spacerun:yes'>&nbsp;&nbsp; </span><span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>1.509693e+01<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>2<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>AGTT<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>1.461676e+01<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>3<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>CTAG<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>1.180616e+01<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>4<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>CCCC<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>1.132821e+01<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>5<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>GGGG<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>1.041196e+01<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>6<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>TATA<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>9.547606e+00<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>7<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>GTTC<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>8.937067e+00<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>8<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>AAGT<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>8.544586e+00<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>9<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>CAGT<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>8.471593e+00<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>10<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>CCCA<span
style='mso-spacerun:yes'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>7.966514e+00<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US style='font-size:7.5pt'>.....<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'>The
first line is sequence name, and all the genomic signatures are sorted
according to their kurtosis scores.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp;&nbsp; </span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'>Output
of the predicted GIs of total scales is a Zip file whose name is created by the
input file name and predict GIs. If input file contains at least two sequences,
the Zip file contains more predict GIs files for all the sequences. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp;&nbsp; </span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'>Here
is a brief description of the predicted GIs of the total scales:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1.
<span class=SpellE><span class=GramE>seqid</span></span> - The name of the
sequence. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>2.
<span class=GramE>source</span> - The program <span class=SpellE>MTGIpick</span><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>3.
type- The name of this type of feature, genomic island. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>4.
<span class=GramE>start</span> - The starting position of the predicted genomic
island in the sequence. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>5.
<span class=GramE>end</span> - The ending position of the predicted genomic
island in the sequence. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>6.
<span class=GramE>score</span> - The conserved scores of each nucleotide
according to the total number of appearances in the predicted genomic island
along total scales in <span class=SpellE>MTGIpick</span>.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>7.
<span class=GramE>strand</span> - The strand of the feature, + for positive
strand.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>8.
<span class=GramE>phase</span> - There’s no annotation information and be
replaced with “.” <span class=SpellE>charcater</span>. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>9.
<span class=GramE>attributes</span> - The numbers of the genomic islands.<span
style='mso-spacerun:yes'>&nbsp;&nbsp; </span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'>Example
the total scale predict GIs looks like this:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>##<span class=SpellE>gff</span>-version 3<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>##sequence-region Example 1 757540<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>125001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>137000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI1<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>266001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>269000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>6<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI2<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>269001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>272000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>7<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI3<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>272001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>287000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>30<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI4<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>287001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>288000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>9<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI5<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>288001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>305000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>30<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI6<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>305001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>306000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>28<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI7<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>328001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>329000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>2<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI8<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>329001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>340000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>3<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI9<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>340001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>342000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI10<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>430001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>431000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI11<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>431001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>447000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>2<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI12<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>Example<span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>546001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>558000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI13<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:7.5pt;
line-height:115%'>.....</span></span></span><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:5.5pt;
line-height:115%'><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp; </span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'><span
style='mso-spacerun:yes'>&nbsp;</span>Output of each scale predict GIs is a Zip
file whose name is created by the input file name and predict. If the input
file contains at least two sequences, the Zip file contains more predicted GI files
for all the sequences. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp; </span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'><span
style='mso-spacerun:yes'>&nbsp;</span>Here is a brief description of the predicted
GIs of each scale:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1.
<span class=SpellE><span class=GramE>seqname</span></span> - The name of the
sequence. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>2.
<span class=GramE>source</span> - The program <span class=SpellE>MTGIpick</span><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>3.
<span class=GramE>type</span> - The name of this type of feature, genomic
island. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>4.
<span class=GramE>start</span> - The starting position of the predicted genomic
island in the sequence. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>5.
<span class=GramE>end</span> - The ending position of the predicted genomic
island in the sequence. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>6.
<span class=GramE>score</span> - The conserved scores of each nucleotide
according to the total number of appearances in the predicted genomic island
along total scales in <span class=SpellE>MTGIpick</span>.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>7.
<span class=GramE>strand</span> - The strand of the feature, + for positive
strand.<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>8.
<span class=GramE>phase</span> - There’s no annotation information and be
replaced with “.” <span class=SpellE>charcater</span>. <o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>9.
<span class=GramE>attributes</span> - the numbers of the genomic islands.<span
style='mso-spacerun:yes'>&nbsp; </span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'><span
style='mso-spacerun:yes'>&nbsp;</span>The predict GIs of each scale looks like
this:<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Scale 1<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>##<span class=SpellE>gff</span>-version
3<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>##sequence-region Example 1
757540<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Example<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>272001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>287000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI1<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Example<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>288001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>306000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI2<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'><o:p>&nbsp;</o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Scale 2<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>##<span class=SpellE>gff</span>-version
3<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>##sequence-region Example 1
757540<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Example<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>272001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>287000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI1<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Example<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>288001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>306000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI2<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'><o:p>&nbsp;</o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Scale 3<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>##<span class=SpellE>gff</span>-version
3<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>##sequence-region Example 1
757540<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Example<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>272001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>287000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI1<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>Example<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>MTGIpick</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp; </span><span
class=SpellE>genomic_island</span><span style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>288001<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>306000<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>1<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>+<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>.<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>ID=GI2<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:115%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:7.5pt;line-height:115%'>.....<span
style='mso-tab-count:1'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:25%;tab-stops:44.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:5.5pt;line-height:25%'><o:p>&nbsp;</o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%'><span style='mso-bookmark:OLE_LINK1'><span
style='mso-bookmark:OLE_LINK2'><span lang=EN-US style='font-size:12.0pt;
line-height:150%;color:blue'>7.2) Interactive visualization tool<o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial;
color:#0C0C0C'><span style='mso-spacerun:yes'>&nbsp; </span></span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-bidi-font-family:Arial'><span
style='mso-spacerun:yes'>&nbsp;</span></span><span class=SpellE><span
lang=EN-US>MTGIpick</span></span><span lang=EN-US> provides a new interactive
genome visualization tool, which uses <span class=SpellE>zoomable</span>
sunburst and sunburst partition to represent predicted GIs with conserved score
along the whole genome. <span class=SpellE>MTGIpick</span> has generated a
number of html files in the same directory where the input file is stored, you
can open them directly and view the predicted GIs with conserved scores.</span></span></span><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='font-size:12.0pt;line-height:150%;mso-font-kerning:0pt'><o:p></o:p></span></span></span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US><!--[if gte vml 1]><v:shapetype id="_x0000_t75" coordsize="21600,21600"
 o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe" filled="f"
 stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="图片_x0020_1" o:spid="_x0000_i1025" type="#_x0000_t75"
 style='width:275.25pt;height:149.25pt;mso-position-horizontal-relative:page;
 mso-position-vertical-relative:page'>
 <v:imagedata src="help.files/image001.jpg" o:href="../../AppData/Local/Temp/ksohtml/wps4C31.tmp.jpg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=367 height=199
src="help.files/image001.jpg" v:shapes="图片_x0020_1"><![endif]><span
style='mso-spacerun:yes'>&nbsp;</span><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US>Orange regions in the first circle represent GIs predicted by <span
class=SpellE>MTGIpick</span> method at all the scales. Click on any orange
region in the first circle to zoom in, conserved scores of this predicted GI
will be represented, where darker red colors are used to represent higher
conserved score; light-to-dark for low-to-high.<span
style='mso-spacerun:yes'>&nbsp; </span><o:p></o:p></span></span></span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US><o:p>&nbsp;</o:p></span></span></span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="图片_x0020_2"
 o:spid="_x0000_i1027" type="#_x0000_t75" style='width:294pt;height:2in;
 visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="help.files/image002.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=392 height=192
src="help.files/image003.jpg" v:shapes="图片_x0020_2"><![endif]></span><span
lang=EN-US><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US>Click on any region of the above predicted GI, more information on
the predicted GI will be represented at the top right hand corner of the page,
such as start position, end position, size, conserved score and GC content.<o:p></o:p></span></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:
OLE_LINK2'><span lang=EN-US><span style='mso-spacerun:yes'>&nbsp;</span><o:p></o:p></span></span></span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US style='mso-no-proof:yes'><!--[if gte vml 1]><v:shape id="_x0000_i1026"
 type="#_x0000_t75" style='width:293.25pt;height:145.5pt;visibility:visible;
 mso-wrap-style:square'>
 <v:imagedata src="help.files/image004.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=391 height=194
src="help.files/image005.jpg" v:shapes="_x0000_i1026"><![endif]></span><span
lang=EN-US><o:p></o:p></span></span></span></p>

<p class=MsoNormal style='line-height:150%;tab-stops:21.6pt'><span
style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
lang=EN-US>Click on the center circle to zoom out. </span></span></span></p>

</div>

</body>

</html>
